src/app/onto-search/onto-search.component.ts

```diff
@@ -0,0 +1,129 @@
+    'labels': ['Duloxetine Hydrochloride', 'Duloxetine HCl'],
+  }, {
+    'uri': {
+      'namespace': 'http://linkedlifedata.com/resource/drugbank/drug/',
+      'localName': 'DB01248',
+    },
+    'label': 'Docetaxel',
+    'type': 'small molecule',
+    'score': 1.1766968,
+    'definition': 'Docetaxel is a clinically well established anti-mitotic chemotherapy medication used mainly for the treatment of breast, ovarian, and non-small cell lung cancer. Docetaxel binds to microtubules reversibly with high affinity and has a maximum stoichiometry of 1 mole docetaxel per mole tubulin in microtubules.',
+    'labels': ['TXL', 'Docetaxel anhydrous', 'Docetaxel, Trihydrate'],
+  }, {
+    'uri': {
+      'namespace': 'http://linkedlifedata.com/resource/drugbank/drug/',
+      'localName': 'DB01609',
+    },
+    'label': 'Deferasirox',
+    'type': 'small molecule',
+    'score': 1.1766968,
+    'definition': 'Deferasirox is an oral iron chelator. Its main use is to reduce chronic iron overload in patients who are receiving long term blood transfusions for conditions such as beta-thalassemia and other chronic anemias. It is the first oral medication approved in the USA for this purpose.',
+    'labels': ['Deferasiroxum [inn-latin]', 'ICL 670'],
+  }, {
+    'uri': {
+      'namespace': 'http://linkedlifedata.com/resource/drugbank/drug/',
+      'localName': 'DB00003',
+    },
+    'label': 'Dornase Alfa',
+    'type': 'biotech',
+    'score': 1.1766968,
+    'definition': 'Dornase alfa is a biosynthetic form of human deoxyribunuclease I (DNase I) enzyme. It is produced in genetically modified Chinese hamster ovary (CHO) cells using recombinant DNA technology. The 260-amino acid sequence of dornase alfa is identical to the endogenous human enzyme. Dornase alfa cleaves extracellular DNA to 5´-phosphodinucleotide and 5´-phosphooligonucleotide end products without affecting intracellular DNA. In individuals with cystic fibrosis, extracellular DNA, which is an extremely viscous anion, is released by degenerating leukocytes that accumulate during inflammatory responses to infections. Enzymatic breakdown of this extracellular DNA appears to reduce sputum viscosity and viscoelasticity.',
+    'labels': ['Deoxyribonuclease I', 'rhDNase', 'DNase I', 'Deoxyribonuclease-1 precursor', 'DNase'],
+  }]);
+
+  states: Observable<SearchModel[]>;
```

Comment:
Isn't this supposed to be an Input? And also have an Output with the autocomplete keyword?